Common/Securities/SecurityCache.cs

```diff
@@ -207,21 +207,16 @@ public void AddData(BaseData data)
         /// Stores the specified data list in the cache WITHOUT updating any of the cache properties, such as Price
         /// </summary>
         /// <param name="data">The collection of data to store in this cache</param>
-        public void StoreData(IReadOnlyList<BaseData> data)
+        /// <param name="dataType">The data type</param>
+        public void StoreData(IReadOnlyList<BaseData> data, Type dataType)
         {
-            if (data.Count == 0)
```

Comment:
interesting... why remove this?

Reply:
Reviewed the consumers and it wasn't a case that could happen today, so I figured I'd remove it